symbols-mui/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @emclaug2 @joebochill @daileytj
```

Comment:
Are you allowed to have a codeowners file nested like this?

Reply:
has to be in the root, although you may technically change that `*` to say `symbols-mui/**` to specify code owners for a particular set of file.

Reply:
I've just removed the duplicated codeowners file and added Tom to the top-level CODEOWNERS